h/static/scripts/directive/annotation.js

```diff
@@ -739,6 +739,14 @@ function AnnotationController(
     return domainModel.updated;
   };
 
+  vm.updatedString = function () {
+    if (!domainModel.updated) {
+      return '';
+    }
+    var date = new Date(domainModel.updated);
+    return date.toDateString() + ', ' + date.toLocaleTimeString();
```

Comment:
`toDateString()` actually generates the most similar output to the previous result. I've replaced this now with `toLocaleDateString()` which provides customization of the output where supported (IE 11, Edge, Firefox, Chrome) and fall back to `toDateString()` elsewhere (IE 10, Safari).

A downside of not using Moment.js is that the output in this case is implementation dependent but I think it is the right trade-off vs. the page weight of Moment.